deploy/helm/sumologic/values.yaml

```diff
@@ -786,9 +786,9 @@ fluent-bit:
 
     @INCLUDE fluent-bit-output.conf
 
-## Configure prometheus-operator
-## ref: https://github.com/helm/charts/blob/master/stable/prometheus-operator/values.yaml
-prometheus-operator:
+## Configure kube-prometheus-stack
+## ref: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml
+kube-prometheus-stack:
```

Comment:
Do we want to use `kube-prometheus-stack`? We could use `alias` in the requirements to keep backward compability in the configuration